ifrs17/Import/Importers.ipynb

```diff
@@ -1369,13 +1369,20 @@
                 "\n                if(values.Length == 0 && !parsingStorage.MandatoryAocSteps.Contains(new AocStep(aocType, novelty))) return null;",
                 "\n            }",
                 "\n            ",
+                "\n            var accidentYearInput = (datarow.Field<string>(nameof(RawVariable.AccidentYear)));",
+                "\n            var isAccidentYearInt = Int32.TryParse(accidentYearInput, out var accidentYear);",
+                "\n            if(hasAccidentYearColumn)",
+                "\n            { ",
+                "\n                if(!(isAccidentYearInt || accidentYearInput == null)) ApplicationMessage.Log(Error.AccidentYearTypeNotValid, accidentYearInput);",
```

Comment:
transform this condition to be a positive condition: 
!isAYint && AYInput != null